src/views/virtualmachines/details/components/details/VirtualMachineDetailsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ListPageBody, ListPageHeader } from '@openshift-console/dynamic-plugin-sdk';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import VirtualMachineDetailsLeftGrid from './components/grid/leftGrid/VirtualMachineDetailsLeftGrid';
+import VirtualMachineDetailsRightGrid from './components/grid/rightGrid/VirtualMachineDetailsRightGrid';
+
+type VirtualMachineDetailsPageProps = RouteComponentProps<{
+  ns: string;
+  name: string;
+}> & {
+  obj?: V1VirtualMachine;
```

Comment:
Why not `vm`?

Reply:
the `HorizontalNav` component returns the resource given to it as obj, I added some alias to make it more clear :)